src/com/superiornetworks/icarus/commands/Command_say.java

```diff
@@ -0,0 +1,27 @@
+package com.superiornetworks.icarus.modules;
+
+import com.superiornetworks.icarus.ICM_Rank;
+import com.superiornetworks.icarus.IcarusMod;
+import org.bukkit.Bukkit;
+import org.bukkit.command.CommandSender;
+import org.bukkit.entity.Player;
+
+public class Command_say extends JavaPlguin 
+{
+    public say(IcarusMod plugin) 
+    {
+        super(plugin);
+    }
+    
+    public static void chat(CommandSender sender, String message)
```

Comment:
This is a command class, not a util class. This needs to be a boolean, not static, you need to add this to plugin.yml, and you need to use the @CommandParameters notation above the class name thingy.